src/docs/index.md

```diff
@@ -205,6 +218,10 @@ title: <qa-bot> ⌲ Home
                         open: undefined,
                         showTip: undefined,
                         tipText: '',
+                        unknownAnswer: {
+                            text: '',
+                            url: ''
+                        },
```

Comment:
Why use a new block of config instead of leveraging the `texts` templates?
```html
<template slot="texts">
    <span for="unknownAnswerText">...</span>
    <span for="unknownAnswerLink">...</span>
</template>
```